src/core/mmu/vmm.c

```diff
@@ -16,14 +16,14 @@ struct vm_install_info vmm_get_vm_install_info(struct vm_allocation* vm_alloc)
 {
     struct vm_install_info info = {
         .base = vm_alloc->base,
-        .vm_section_pte = *pt_get_pte(&cpu()->as.pt, 0, vm_alloc->base),
+        .vm_section_pte = *pt_get_pte(&cpu()->as.pt, VM_SHARED_PT_LVL, vm_alloc->base),
```

Comment:
On file  `src/core/mmu/mem.c` , shouldn't the parameter on the functions `pt_get_pte` (line  810/812) be changed to use `VM_SHARED_PT_LVL` ?

Reply:
I think this was already fixed by another PR. It was "fixed" after a rebase